rules/linux/credential_access_potential_linux_ssh_bruteforce_root.toml

```diff
@@ -17,6 +17,33 @@ index = ["auditbeat-*", "logs-system.auth-*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Potential SSH Brute Force Detected on Privileged Account"
+note = """## Triage and analysis
+
```

Comment:
Should we add `### Investigating Potential SSH Brute Force Detected on Privileged Account` as a header here to remain consistent with the flow of other investigation guides?